velox/docs/functions/presto/datetime.rst

```diff
@@ -95,6 +95,22 @@ Date and Time Functions
 .. function:: from_iso8601_date(string) -> date
 
     Parses the ISO 8601 formatted ``string`` into a ``date``.
+    ISO 8601 ``string`` can be formatted as any of the following:
+    ``[+-][Y]Y*``
+    ``[+-][Y]Y*-[M]M``
+    ``[+-][Y]Y*-[M]M*-[D]D``
+    ``[+-][Y]Y*-[M]M*-[D]D *``
+
+    Year value must contain at least one digit, and may contain up to six digits.
+    Month and day values are optional and may each contain one or two digits.
+
+    Examples of supported input strings:
```

Comment:
Would you generate the docs locally and double check they render nicely?

Reply:
Good call, the new lines were wonky - fixed it. Also fixed some missing asterisks in the format syntax I noticed.